README.md

```diff
@@ -11,14 +11,13 @@
 ### ![Lab 6 WRRC](lab6_wrrc.png)
 ### ![Lab 7 WRRC](lab7_wrrc.png)
 ### ![Lab 8 WRRC](lab7_wrrc.png)
+### ![Lab 9 WRRC](lastWRRC.png)
+### ![Lab 10 WRRC](lastWRRC.png)
```

Comment:
These images aren't showing as you need to add the file path (where they live) 👍 

So: ![Lab 10](./where it lives/name of image). This should do the trick!